core/admin/admin.py

```
@@ -0,0 +1,320 @@
+    """Admin view for Backup Users (fallback mode)"""
+    name = "Backup User"
+    name_plural = "Backup Users"
+    icon = "fa-solid fa-database"
+    
+    column_list = [
+        BackupUser.id,
+        BackupUser.mobile_number,
+        BackupUser.hash_value,
+        BackupUser.restored,
+        BackupUser.created_at,
+    ]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)